cdbsearch.py

```diff
@@ -411,41 +413,46 @@ async def search(self, board, depth):
         tasks = {}
         tried_unscored = False
 
-        for move in board.legal_moves:
-            ucimove = move.uci()
-            score = scored_db_moves.get(ucimove, None)
-            newdepth = self.move_depth(bestscore, worstscore, score, depth)
+        # guarantee sufficient length of the executorTree list
```

Comment:
comment should read `semaphoreTree`

Reply:
in addition, comment could say something like: "limit concurrent tasks per level of the search tree" or similar?